### PR TITLE
Additions to the Web page (navbar, project page, row that displays basic info on main page)

### PR DESCRIPTION
**Pull request by GrantComm**
Additions to the Web page (navbar, project page, row that displays basic info on main page)

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -2,17 +2,69 @@
...
+          mobile developent. 
+      </p>
+      <p>Click here to get a random fact about me</p>
+      <button onclick="addRandomFact()">Click Me!</button>
       <div id="greeting-container"></div>
-    </div>
+      <div>
+        
+      </div>
+      </div>
```
Comment (acmcarther) [inline portfolio/src/main/webapp/index.html:53]: Indentation is funny here (dedent one level)

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -2,17 +2,69 @@
...
+    <div id="intro">
+        <div id="innerIntro">
+            <h1 style="font-size: 65px; margin: 0">Grant Commodore</h1>
+            <h2 style="margin:0">Sophmore Computer Science Major, Mathematics Minor</h2>
+            <h2 style="margin:0">Aspiring Software Engineer</h2>
+        </div>
+    </div>
+
+    <div class="row">
+        <div class="column">
```
Comment (acmcarther) [inline portfolio/src/main/webapp/index.html:27]: Consider using a consistent spacing (either 4sp or 2sp)

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -2,17 +2,69 @@
...
+        <a class="active" href="index.html">About Me</a>
+        <a href="projects.html">Projects</a>
+        <a href="#skills">Skills</a>
+        <a href="#contact">Contact</a>
+    </div>
+
+    <div id="intro">
+        <div id="innerIntro">
+            <h1 style="font-size: 65px; margin: 0">Grant Commodore</h1>
+            <h2 style="margin:0">Sophmore Computer Science Major, Mathematics Minor</h2>
```
Comment (acmcarther) [inline portfolio/src/main/webapp/index.html:21]: Optional: you can probably drop these inline styles by styling children of `innerIntro`. Works best if you give the outer div a class (i.e. class="innerIntro")

I think the syntax is:
```css
.innerIntro > h1 {
  margin: 0
}
.innerIntro > h2 {
  margin: 0
}
```

File: portfolio/src/main/webapp/index.html [hunk 4]
```diff
@@ -2,17 +2,69 @@
...
+      <p style="text-align:center;">Hello there! My name is Grant Commodore and I am a sophmore computer
+          science major, mathematics minor at Morehouse College from PG 
+          County Maryland. I am an aspiring software engineering with an interest in backend and frontend
+          mobile developent. 
+      </p>
+      <p>Click here to get a random fact about me</p>
+      <button onclick="addRandomFact()">Click Me!</button>
       <div id="greeting-container"></div>
-    </div>
+      <div>
```
Comment (acmcarther) [inline portfolio/src/main/webapp/index.html:50]: what's this div?

File: portfolio/src/main/webapp/projects.html [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+    </div>
+
+    <div id="content">
+      <h1 style="text-align:center;">Projects</h1>
+      <p style="text-align:center;">These are some of the projects that I have been able to work on</p>
+      <p>Click here to get a random fact about me</p>
+      <button onclick="addRandomFact()">Click Me!</button>
+      <div id="greeting-container"></div>
+      </div>
+
```
Comment (acmcarther) [inline portfolio/src/main/webapp/projects.html:25]: rm extra newline

File: portfolio/src/main/webapp/projects.html [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+      <p>Click here to get a random fact about me</p>
+      <button onclick="addRandomFact()">Click Me!</button>
+      <div id="greeting-container"></div>
+      </div>
+      
+  </body>
+
+  <footer>
+      <div id="contact"> 
+        <h2> Contact Me </h2>
```
Comment (acmcarther) [inline portfolio/src/main/webapp/projects.html:30]: dedent?

File: portfolio/src/main/webapp/script.js [hunk 7]
```diff
@@ -15,10 +15,13 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
+function addRandomFact() {
+  const greetings = [
```
Comment (acmcarther) [inline portfolio/src/main/webapp/script.js:19]: Might want to rename this variable

File: portfolio/src/main/webapp/index.html [hunk 8]
```diff
@@ -2,17 +2,47 @@
...
+        </div>
+    </div>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+        <p style="text-align:center;">
+            Hello there! My name is Grant Commodore and I am a sophmore computer
```
Comment (jalexanderqed) [inline portfolio/src/main/webapp/index.html:38]: r/sophmore/sophomore

File: portfolio/src/main/webapp/projects.html [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+    </div>
+    <div id="content">
+        <h1 style="text-align:center;">Projects</h1>
+        <p style="text-align:center;">These are some of the projects that I have been able to work on</p>
+        <p>Click here to get a random fact about me</p>
+        <button onclick="addRandomFact()">Click Me!</button>
+        <div id="greeting-container"></div>
+    </div>  
+  </body> 
+</html>
```
Comment (jalexanderqed) [inline portfolio/src/main/webapp/projects.html:23]: Newline at end of this file

File: portfolio/src/main/webapp/style.css [hunk 10]
```diff
@@ -2,8 +2,80 @@
...
+.topnav a:hover {
+  background-color: #ddd;
+  color: black;
+}
+
+/*Changes the color of the active item in the navbar*/
+.topnav a.active {
+  background-color: rgb(231, 131, 16);
+  color: white;
+}
```
Comment (jalexanderqed) [inline portfolio/src/main/webapp/style.css:81]: Newline at end of this file.

File: portfolio/src/main/webapp/index.html [hunk 11]
```diff
@@ -1,18 +1,125 @@
...
+<head>
+	<meta charset="UTF-8">
+	<title>Grant Commodore's Portfolio Project</title>
+	<link rel="stylesheet" href="style.css">
+	<script src="script.js"></script>
+</head>
+
+<body>
+
+	<div id="nav" , class="topnav">
```
Comment (acmcarther) [inline portfolio/src/main/webapp/index.html:13]: Extreme nit: rm space in front of comma

(apply below as well)

File: portfolio/src/main/webapp/index.html [hunk 12]
```diff
@@ -1,18 +1,125 @@
...
+
+	<!-- The Modal -->
+	<div id="myModal" class="modal">
+		<!-- Modal content -->
+		<div class="modal-content">
+			<div class="modal-header">
+				<h2 id="modalHeader">Modal Header</h2>
+			</div>
+			<div class="modal-body">
+				<img id= "modalImage"src="">
```
Comment (acmcarther) [inline portfolio/src/main/webapp/index.html:27]: Extreme nit: add space between "modalImage" and src

File: portfolio/src/main/webapp/index.html [hunk 13]
```diff
@@ -1,18 +1,125 @@
...
+
+	<div id="content">
+		<h2 class="sectionHeader" , style="padding-top:10px">Connect with Me</h2>
+		<p style="text-align:center; font-size:19px;">
+			Thank you for taking to time to learn a little bit about me!
+			If you have any questions, please feel free to connect on LinkedIn.
+		</p>
+		<button class="button", onclick="window.location.href='https://www.linkedin.com/in/grant-commodore-753b98171';">Let's Connect!</button>
+		<div id="fact-container"></div>
+		<div style="text-align:left">Icons made by <a href="https://www.flaticon.com/authors/freepik"
```
Comment (acmcarther) [inline portfolio/src/main/webapp/index.html:119]: This looks kind of funny (inlined <a>) but I'm not sure what you should do to clean this up. @jalexanderqed any suggestions?

File: portfolio/src/main/webapp/script.js [hunk 14]
```diff
@@ -15,14 +15,122 @@
...
+    const fact = facts[Math.floor(Math.random() * facts.length)];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+    return fact;
+}
+
+function showModal(typeModal) {
+
+    //Variables for the about me row
```
Comment (acmcarther) [inline portfolio/src/main/webapp/script.js:35]: Nit: style guide advises space between "//" and the start of the comment text.

File: portfolio/src/main/webapp/style.css [hunk 15]
```diff
@@ -1,9 +1,368 @@
...
+  border-radius: 5px;
+}
+
+#funFactsButton:hover {
+  background-color: rgb(244, 247, 244);
+  color: rgb(231, 131, 16);
+}
+
+#mobileButton {
+  background-color: rgb(231, 131, 16);
```
Comment (acmcarther) [inline portfolio/src/main/webapp/style.css:237]: Optional: I think you can declare this rgb value as a variable and reuse it everywhere.

See https://www.w3schools.com/css/css3_variables.asp

**Pull request merged**